gradle/formatting.gradle

```diff
@@ -22,5 +22,10 @@ allprojects {
                 paddedCell()
             }
         }
+        format("license", {
+            licenseHeaderFile("${rootProject.file("formatter/license-header.txt")}", "package ");
+            target("src/main/java/**/*.java")
+            targetExclude("**/netty4/*")
```

Comment:
Netty4 package has the licenses already for all the files, so let's not exclude it.

Reply:
> Netty4 package has the licenses already for all the files, so let's not exclude it.

Netty4 has dual headers.  Try removing that exclusion and running spotless.  It removes the whole second license header underneath the SPDX one.   I'm sure there's probably some way to configure it to avoid it doing that, but that'll take more time to figure out.  

![image](https://user-images.githubusercontent.com/9291703/179078813-a2f56bdd-d538-466b-8d4f-f6f00169ab63.png)

Reply:
Okay. So let's exclude netty4 package because we don't want to remove ElasticSearch license from those files.

Reply:
Looked into this just a bit longer.  The configuration includes a delimiter (in this case "package ") to mark the end of the license, which is what caused the above behavior.  And there's no obvious alternative delimiter for the netty4 files that we could add that wouldn't fail on the other files (unless maybe it's a regex but that's getting even more complicated.)

I think leaving it as-is and providing the option to exclude (as we've done for netty) is the right way forward.

Reply:
Ideally we should be pulling in these dependencies from OpenSearch. We do have an open issue for it[1]
Longterm this code should go away and I am ok to ignore license checks for `Netty4`. 

[1] https://github.com/opensearch-project/OpenSearch/issues/3118